Source/WebCore/PAL/pal/graphics/WebGPU/Impl/WebGPUAdapterImpl.cpp

```diff
@@ -171,24 +179,23 @@ void AdapterImpl::requestDevice(const DeviceDescriptor& descriptor, CompletionHa
         .minUniformBufferOffsetAlignment =    256,
         .minStorageBufferOffsetAlignment =    256,
         .maxVertexBuffers =    8,
-        .maxBufferSize =    1024 * 1024,
+        .maxBufferSize =    268435456,
         .maxVertexAttributes =    16,
         .maxVertexBufferArrayStride =    2048,
-        .maxInterStageShaderComponents =    32,
-        .maxInterStageShaderVariables =    32,
+        .maxInterStageShaderComponents =    60,
+        .maxInterStageShaderVariables =    16,
         .maxColorAttachments =    8,
-        .maxComputeWorkgroupStorageSize =    16352,
+        .maxColorAttachmentBytesPerSample = 32,
+        .maxComputeWorkgroupStorageSize =    16384,
         .maxComputeInvocationsPerWorkgroup =    256,
         .maxComputeWorkgroupSizeX =    256,
         .maxComputeWorkgroupSizeY =    256,
         .maxComputeWorkgroupSizeZ =    64,
         .maxComputeWorkgroupsPerDimension =    65535,
     };
 
-    auto requestInvalidDevice = [this, &callback]() {
-        wgpuAdapterRequestInvalidDeviceWithBlock(m_backing, makeBlockPtr([protectedThis = Ref { *this }, convertToBackingContext = m_convertToBackingContext.copyRef(), callback = WTFMove(callback)](WGPUDevice device) mutable {
-            callback(DeviceImpl::create(device, Ref { protectedThis->features() }, Ref { protectedThis->limits() }, convertToBackingContext));
-        }).get());
+    auto requestInvalidDevice = [&callback]() {
+        callback(nullptr);
     };
```

Comment:
Is this really worth it? If it were me, I'd just inline this lambda at call sites.